addons/knobs/src/react/WrapStory.js

```diff
@@ -45,6 +45,10 @@ export default class WrapStory extends React.Component {
     const { knobStore, storyFn, context } = this.props;
     // Update the related knob and it's value.
     const knobOptions = knobStore.get(name);
+
+    // if the knob is a button, a change event should call the callback
+    if (knobOptions.callback) knobOptions.callback();
```

Comment:
Let's use a separate method for that. There're no "changes" here actually

Reply:
Fair point. I added click handlers that live alongside the change handlers.
Both `onClick` and `onChange` are passed to each `InputType` now, but it's up to that particular `InputType` (in this case, `ButtonType`) to decide how to use each.